app/views/admin/tickets/show.html.haml

```diff
@@ -6,6 +6,11 @@
         Ticket
         %small
           = humanized_money_with_symbol @ticket.price
+      - if @ticket.description?
```

Comment:
This is potentially very large. Make this a [tooltip or popover](http://getbootstrap.com/javascript/#tooltips) on `@ticket.title`